lib/utils/computed-property.js

```diff
@@ -29,7 +32,6 @@ function isCPDesc(node, method) {
   return false; // don't know how you could get here
 }
 
-
 function isPrototypeExtCP(node) {
```

Comment:
Prefer full name. `isPrototypeExtComputedProperty`. What does the `ext` part represent?